robocop/checkers/naming.py

```diff
@@ -242,6 +243,17 @@
         msg="Variable '{{ variable_name }}' may overwrite similar variable inside '{{ block_name }}' {{ block_type }}. "
         "Note that variables are case-insensitive, and also spaces and underscores are ignored.",
         severity=RuleSeverity.INFO,
+        docs="""
+        Following assignments overwrite the same variable::
+        
+            *** Keywords ***
+            Keyword
+                ${variable}    Keyword Call
+                ${VARIABLE}    Keyword Call
+                ${vari_ab le}    Keyword Call
+        
+        Use consistent variable naming guidelines to avoid not intended variable overwriting.
```

Comment:
```suggestion
        Use consistent variable naming guidelines to avoid unintended variable overwriting.
        Remember that variable names in Robot Framework are case-insensitive and
        underscores and whitespaces are ignored.
```